docs/faq.md

```diff
@@ -0,0 +1,89 @@
+
+## Can I see a sample of each format?
+
+Go to [our demo folder](https://github.com/mwouts/jupytext/tree/master/demo) and see how our sample `World population` notebook is represented in each format.
+
+## Can I edit the paired text file?
+
+Yes! And when you're done, refresh the notebook in Jupyter. Refreshing will bring the latest changes to your notebook.
+
+## How do paired notebooks work?
+
+The `.ipynb` file contains the full notebook. The paired text file only contains the input cells and selected metadata. When the notebook is loaded by Jupyter, input cells are loaded from the text file, while the output cells and the filtered metadata are restored using the `.ipynb` file.
+
+## Can I create a notebook from a text file?
+
+Certainly. Open your pre-existing scripts or Markdown files as notebooks with a click in Jupyter Notebook, and with the _Open as Notebook_ menu in JupyterLab. Pair these document to an `.ipynb` file if you want to see the outputs preserved when you refresh the notebook.
+
+If you prefer to do that programmatically, use one of
+```bash
+jupytext --to ipynb *.md                        # convert all .md files to notebooks with no outputs
+jupytext --to ipynb --execute *.md              # convert all .md files to notebooks and execute them
+jupytext --set-formats ipynb,md --execute *.md  # convert all .md files to paired notebooks and execute them
+```
+
+## Which files should I version control?
+
+Unless you want to version control the output cells, you should version the text file only (and add `*.ipynb` to `.gitignore`). As discussed above, Jupyter will let you open the text representation as a notebook and will re-create the `.ipynb` file when you save the notebook.
+
+Note that if you version control both the `.md` and `.ipynb` files, you can configure `git diff` to [ignore the diffs on the `.ipynb` files](https://github.com/mwouts/jupytext/issues/251).
+
+## Jupyter warns me that the file has changed on disk
+
+By default, Jupyter saves your notebook every 2 minutes. Fortunately, it is also aware that you have edited the text file.
+
+You can deactivate the autosave in Jupyter with the Jupytext Menu in Jupyter Notebook, and with the _Autosave Document_ setting in JupyterLab. See [here](https://stackoverflow.com/questions/25631344/turn-off-autosave-in-ipython-notebook/56549758#56549758) if you want to permanently deactivate the autosave in Jupyter Notebook.
+
+## When I refresh, Jupyter warns me that my notebook has unsaved changes
+
+Oh - you have edited both the notebook and the paired text file at the same time? Backup the text file (`git stash`), save the notebook, and merge your changes on the text file (`git stash pop`). When you're done, refresh the notebook in Jupyter.
```

Comment:
IMO this is going to be quite an advanced technique for most users - is there a simpler way to resolve this?

Reply:
> is there a simpler way to resolve this?

None other than using a good editor, I am afraid! With PyCharm (mentionned at the next line), you can compare the diffs between memory and disk changes, that is very convenient. 

But this is just a corner case - I think people will notice that they are changing the notebook in two different editors at the same time?